blue-green-upgrade/cluster_ops.sh

```diff
@@ -264,7 +264,7 @@ main() {
 
   # Source the properties file
   if [ -f "${REPO_HOME}/.env" ] ; then
-    # shellcheck source=.env
+    # shellcheck source=.env disable=SC1091
```

Comment:
`# shellcheck source=.env disable=SC1091`

should be

`# shellcheck disable=SC1091`

if the source does not work

Reply:
Both exceptions are needed for a clean `make lint`.  The `.env` file is made and modified by the user manually, so it doesn't exist on a "clean" repo in each of the subfolders.